CharacterCreation/03_Feats.txt

```diff
@@ -689,113 +696,182 @@ now working with. This feat can only be taken once per weapon.
+	also increase R3 to match the new R2.
+12:	May use Weapons - Heavy (Cha) in place of Intimidate when out 
+	of combat whie brandishing a bigger weapon than the target.
+24:	Heavy weapons no longer recieve the Accuracy penalty associated 
+	with being improvised melee weapons while being used as melee weapons.
+36:	+1 Accuracy against targets that are within 2m of at least one 
+	other enemy.
+48:	+10m to Range R3.
+60:	Machine Guns when autofiring, Flamethrowers, Lightning Guns, Antimaterial 
+	Guns firing explosive shot, etc. can be fired at 1m radius areas instead 
+	of a particular target, hitting all characters in the area automatically 
+	unless they succeed on a Reflex save with DC equal to your weapon accuracy 
+	minus applicable cover bonuses, multiplied by 10. Autofire weapons can only 
+	hit as many targets as they fire bullets.
+
+== Explosives Expertise ==
+
+	Explosives Expertise applies to hand grenades, emplaced explosives, 
+grenade launchers, rocket launchers, and artillery, but not to explosive ammo of 
+other weapons or class abilities. Does apply to Engineer mortars.
+
+0:	+10m to range R2. If this would cause R2 to be greater than R3, 
+	also increase R3 to match the new R2.
+12:	May use Weapons - Explosives (Int) in place of Knowledge checks when analyzing 
+	structural integrity/weaknesses of mechanical systems, such as structures, 
+	vehicles, or robots.
+24:	Reduce the critical failure chance for cooking hand grenades and setting 
+	anti-tamper explosives addons to 2%.
+36:	Reduce Accuracy penalties for indirect fire by 2.
+48:	+5m to range R1. If this would cause R1 to be greater than R2, 
+	also set R2 equal to the new R1.
+60:	+10% Base damage. Other characters make Shock, Reflex, and Will saves associated 
+	with your explosives with Disadvantage.
+
+== Melee Expertise ==
+
+	Applies to Melee Weapons that the user has Proficiency with and Unarmed Strike. 
+Does not apply to Improvised weapons. Reminder: these bonuses are only in effect while 
+you are wielding an appropriate weapon.
+
+0:	+1 Move Speed while using the Charge action with intent to engage an enemy in 
+	Melee Combat.
+12:	May use Weapons - Melee (Str/Dex) in place of Acrobatics or Athletics on 
+	maneuver checks to avoid, gain control of, or escape a Grapple; defend against 
+	Disarm attempts; or defend against Trip attempts.
+24:	While wielding a Light or One-handed non-Hybrid weapon, whenever an enemy attacks 
+	you in Melee Combat and ties your Guard, you may make a melee attack against that 
+	enemy as a Reaction. 
+	While wielding a Hybrid or Two-handed weapon, you may choose to gain +2 to melee 
+	Accuracy checks for one round, but if you do then until the start of your next 
+	turn you take -2 to Guard checks.
+	If dual-wielding a Hybrid and Light or One-handed weapon, you may only use one 
+	of these effects per round, but may choose which.
+36:	+1 Accuracy when engaging a target wielding a melee weapon (unarmed counts).

```

Comment:
"including Unarmed"?

Reply:
Clarity: Is this just for the round when you engage the target, or is it always applicable while engaged with a target?

Reply:
It's always on. Would a simple swap of when -> while be a good fix?

Reply:
Ja